scada-lts-e2e-page/scada-lts-e2e-page-core/src/main/java/org/scadalts/e2e/page/core/criterias/identifiers/NodeCriteriaWithNode.java

```diff
@@ -12,16 +12,16 @@
 
     private final Tag parent;
     private final Tag kid;
```

Comment:
I think it's better to use name "child" instead of "kid", as it's more commonly used in programming